### ISSUE TITLE
[Docs] the math is wrong 

### ISSUE THREAD

**Issue by treeshateorcs** [modularml/mojo#1788]
### Where is the problem?

https://www.modular.com/max/mojo

### What can we do better?

![image](https://github.com/modularml/mojo/assets/60894565/f5d3f314-2aa5-4508-a9fb-720819df330c)

0.03 is less than ten times less than 0.20, 6.66 times actually. so, the speedup should be 33300x at best, not 68000x

**Comment by treeshateorcs** [comment]
```

0.20          0.03
-------   = ---------
5000           x


x = 33333
```

hope it's clear now

**Comment by iamtimdavis** [comment]
Thanks @treeshateorcs - great catch, and this was an honest mistake on this page.

When we first published the numbers we had it on this first post (https://www.modular.com/blog/how-mojo-gets-a-35-000x-speedup-over-python-part-1) but by the time we got to the 3rd post (https://www.modular.com/blog/mojo-a-journey-to-68-000x-speedup-over-python-part-3) we just didn't update the numbers correctly on this table.

We've now done that and appreciate you reporting this.
